tests/spdx/model/test_version.py

```diff
@@ -1,4 +1,7 @@
 # SPDX-FileCopyrightText: 2022 spdx contributors
+# SPDX-FileCopyrightText: 2023 2023 spdx contributors
```

Comment:
I don't think that we need to add another copyright here.

Reply:
fixed